src/main.py

```diff
@@ -11,7 +14,10 @@
 class MainWindow(tb.Window):
     def __init__(self):
         super().__init__(themename="superhero")
-        self.title("FITS Image Viewer")
+        constants.load_images()  # load all images once tk is up
+
+        self.title("EMU Viewer")
+        self.iconphoto(True, constants.ICONPNG)
```

Comment:
`default=True` here means that the icon will get passed to all children of this app